beacon/headers/header-content-type.html

```diff
@@ -34,7 +34,11 @@
   promise_test(function(test) {
     assert_true(navigator.sendBeacon(testUrl, what), "SendBeacon Succeeded");
     return pollResult(test, id) .then(result => {
-      assert_true(result.startsWith(contentType), "Correct referrer header result");
+      if (contentType.length == 0) {
```

Comment:
As Anne commented on the bug:
""
Generally Content-Type's value is always a value that is completely known, other than the boundary parameter of multipart/form-data. It seems you could be much more strict overall here.
"""

Can we use assert_equals() in all cases and startsWith() only when the parameter is a FormData?